### ISSUE TITLE
AI: THE SOMNIUM FILES - nirvanA Initiative (1449200)

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Proton#5931]
# Compatibility Report
- Name of the game with compatibility issues: AI: The Somnium Files -nirvanA Initiative (original AI has this issue too, I think.)
- Steam AppID of the game: 1449200

## System Information
Steam Deck, baybee
Also tested on my Desktop running Arch 

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

Only gonna put the relevant part of the logs:

```
winegstreamer error: decodebin1: Your GStreamer installation is missing a plug-in.
winegstreamer error: decodebin1: ../src-gst_base/gst/playback/gstdecodebin2.c(4719): gst_decode_bin_expose (): /GstBin:bin1/GstDecodeBin:decodebin1:
no suitable plugins found:
Couldn't set avdec_h264-0 to PAUSEDCouldn't set avdec_valve_h264-0 to PAUSED
```

## Symptoms 

So, okay, here's the real reason I'm opening this issue: This only seems to impact *some* users. I have been harassing random game journalists on Twitter (and people over @ the AI Steam discussions) and it works perfectly fine on most people's steam decks. 

But... It doesn't on mine. I get the colored bars screen (since the video cannot be played). I've tried everything. I have used several versions of Proton and Proton GE (the latest GE displays it, but distorted). I've used Experimental. I even wiped my Steam Deck, made a new user, logged into the Deck with that user, then family shared the game and logged out of my (main) user so that nothing was ever installed by me with that user.

Still no dice (the wipe was from installation media, too!)

The install on the wiped device was before installing *anything* else, too.

[Here](https://twitter.com/Theswweet/status/1540469968998846464) is a link from another user for exactly what it looks like.

 
## Reproduction

Above answers that too. More than happy to provide ANY info. I'd love to dig deeper but I've already spent several hours on this and it'd be easiest if I just had a deck that it works on.

**Comment by ghost** [comment]
hey! just a heads up, it works fine for me now (after resetting my switch a second time this morning, it pulled a shader update and is using proton 7.0-3)

no idea why it wasn't working earlier when I tried on multiple devices, but *shrug*

**Comment by dominalien** [comment]
This now works for me, too. I switched from GE-Proton (where the movies were garbled) to 7.0-3, deleted Proton files from the Dev menu and now they display ok. Yay :-)

**Comment by ItsRogueRen** [comment]
This game is weird, I'm having an issue on Steam Deck where X-Ray mode just show a white screen. However, this is only the case when playing in Game Mode. If I launch the game from desktop mode, everything works perfectly. Even in the new BPM which is just Deck UI, it has no issues. It's ONLY in game mode, regardless of what launcher settings I use. 

I've found 2 others on Reddit who have had this issue and that's it, and I've tried Proton 7, Experimental, and even GE 7-21 and GE 7-48, all to no avail.

**Comment by misyltoad** [comment]
@ItsRogueRen Hello, just checking this is the X-Ray mode I should be looking for? (Sorry, I am not very familiar with this game)

![image](https://user-images.githubusercontent.com/21316711/217400372-b729f12c-5b9f-4126-9a92-70bcb9cc832f.png)

(I am running the game in Gamescope on my Desktop here to try and repro but it seems to work there, I will test on Deck next)

**Comment by ItsRogueRen** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5931#issuecomment-1421717089

Yes that's it, it seems to only happen in Game Mode on Steam Deck. I can have the X-Ray working just fine on Deck from Desktop mode or on my Linux desktop PC.

**Comment by misyltoad** [comment]
Okay, I tried Gamescope embedded as well on my desktop and it worked fine there too. Interesting.

I will try on Deck now, and see what I find.

**Comment by misyltoad** [comment]
Not able to reproduce here... Does it only happen at certain sections of the game?

![image](https://user-images.githubusercontent.com/21316711/217402566-7ec35e98-7bef-423e-9a5f-795122339b34.png)

Maybe you could send a save file?

**Comment by ItsRogueRen** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5931#issuecomment-1421733985

It's happened to me at every point in the game where X-ray is used. If I load the same save file in desktop mode, suddenly it works as intended.

**Comment by misyltoad** [comment]
@ItsRogueRen I am using SteamOS Main right now (in the advanced channels), can you confirm to me which version of SteamOS you are using?

**Comment by ItsRogueRen** [comment]
> @ItsRogueRen I am using SteamOS Main right now (in the advanced channels), can you confirm to me which version of SteamOS you are using?

I'm on the stable channel, v3.4.4

**Comment by misyltoad** [comment]
Sorry, I still cannot reproduce the issue on Stable (3.4.4) . I tried in both fullscreen and windowed mode.

![image](https://user-images.githubusercontent.com/21316711/217405531-4cfe7ed5-eafe-4cc0-a813-4b5e4d77edea.png)

**Comment by ItsRogueRen** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5931#issuecomment-1421793597

This is what I get in game mode

![IMG_20230207_195942.jpg](https://user-images.githubusercontent.com/75085610/217409850-01a6d1bf-db72-421a-adac-760bc1275154.jpg)

![Screenshot_20230207_211412](https://user-images.githubusercontent.com/75085610/217420250-edc22e6f-d0d2-4f94-879e-c39711338eda.png)

**Comment by ItsRogueRen** [comment]
> @ItsRogueRen I am using SteamOS Main right now (in the advanced channels), can you confirm to me which version of SteamOS you are using?

I'd be happy to share my save file, but I cannot for the life of me find where it's saved... Searching online has been fruitless for this as all I get is other game's save locations (that don't work for this game) or random spoilers for the game I want to avoid

**Comment by ItsRogueRen** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/5931#issuecomment-1421733985

@Joshua-Ashton 
I have manged to find what I think are the files for save data. Extract these to `~/.steam/steam/steamapps/compatdata/1449200/pfx/drive_c/users/steamuser/AppData/Local/SpikeChunsoft/AI Nirvana Initiative/` after making a backup of your versions
[ai-nirvana-save.zip](https://github.com/ValveSoftware/Proton/files/10681717/ai-nirvana-save.zip)
